packages/messaging-api-messenger/src/MessengerClient.ts

```diff
@@ -426,6 +426,49 @@ export default class MessengerClient {
       .then(res => res.data, handleError);
   }
 
+  /**
+   * User Level Menu
+   *
+   * https://developers.facebook.com/docs/messenger-platform/send-messages/persistent-menu#user_level_menu
+   */
+  getUserLevelMenu(
```

Comment:
This should be `getCustomUserSetting`. The `persistent_menu` setting is one of the custom user settings.